core-bundle/src/Picker/AbstractTablePickerProvider.php

```diff
@@ -136,7 +137,12 @@ public function supportsContext($context): bool
         $this->framework->initialize();
         $this->framework->createInstance(DcaLoader::class, [$table])->load();
 
-        return $this->getDataContainer() === $GLOBALS['TL_DCA'][$table]['config']['dataContainer']
+        if (!class_exists($GLOBALS['TL_DCA'][$table]['config']['dataContainer']) || !class_exists($this->getDataContainer())) {
+            trigger_deprecation('contao/core-bundle', '4.9', 'The usage of a non fully qualified class name as DataContainer name has been deprecated and will no longer work in Contao 5.0. Use the fully qualified class name instead, e.g. Contao\DC_Table::class.');
+        }
+
+        return ($this->getDataContainer() === $GLOBALS['TL_DCA'][$table]['config']['dataContainer']
+                || is_a(DataContainer::getDriverForTable($table), $this->getDataContainer(), true))
```

Comment:
```suggestion
        return ($this->getDataContainer() === $GLOBALS['TL_DCA'][$table]['config']['dataContainer']
                || $this->getDataContainer() === DataContainer::getDriverForTable($table))
```

Reply:
Where are the deprecations when not using a FQCN for the DCA driver?

Reply:
uhm, apparently there is none, but it should be here? Otherwise people would not get notified in Contao 4? https://github.com/contao/contao/blob/4.13/core-bundle/src/Resources/contao/classes/DataContainer.php#L1625-L1628

Reply:
Unfortunately, my patch does not work, because `DataContainer::getDriverForTable` returns `DC_Table` if the driver is `Table`, it does not return the actually FQCN 😞